apps/console/src/features/connections/api/authenticators.ts

```diff
@@ -250,6 +250,40 @@ export const getMultiFactorAuthenticatorDetails = (id: string): Promise<MultiFac
         });
 };
 
+/**
+ * Hook to get multi factor authenticatro details.
+ *
+ * @param filter - Search filter.
+ * @param type - Authenticator Type.
+ *
+ * @returns Response as a promise.
+ */
+export const useMultiFactorAuthenticatorDetails = <
```

Comment:
Let's move this to a dedicated default module. `use-multi-factor-authenticator-details.ts`.
This can be done in a later PR.

Reply:
https://github.com/wso2/identity-apps/pull/4889